Makefile

```diff
@@ -139,6 +139,8 @@ func-test:
 check-env:
 	@./scripts/check-env.sh
 
+codespell:
+	codespell --skip=.git,./vendor,./docs/design/proposals/images -L ExtraVersion,extraversion,ba
```

Comment:
Move all these configurations to a configfile https://github.com/codespell-project/codespell#using-a-config-file?

Reply:
Yes I have checked that, but running codespell with config flag shows error: `unrecognized arguments: --config=`

Reply:
I guess the argument parsing of codespell isn't that great. There seems to be a `--config` option, but without the `=`:
```
$ codespell -h | grep config
                 [-A LINES] [-B LINES] [-C LINES] [--config CONFIG]
  --config CONFIG       path to config file.
```

You might be able to use `--config scripts/codespell.conf`

Reply:
oh, okay. Thanks.

Reply:
I am not sure which version you are using @nixpanic  but for me it still fails.
`[yatipadia@192 ceph-csi]$ codespell -h | grep config
[yatipadia@192 ceph-csi]$ 
`

Reply:
```
[🎩︎]mrajanna@localhost volume-replication-operator $]codespell -h |grep config
                 [-A LINES] [-B LINES] [-C LINES] [--config CONFIG]
  --config CONFIG       path to config file.
[🎩︎]mrajanna@localhost volume-replication-operator $]codespell --version
2.0.0
```
latest is 2.1.0 https://github.com/codespell-project/codespell/releases/tag/v2.1.0

Reply:
we verified things a little more, and the next update to this PR will use the `--config` option :smiley:

Reply:
Done now.